packages/mdc-drawer/mdc-drawer.scss

```diff
@@ -20,6 +20,108 @@
 // THE SOFTWARE.
 //
 
-@import "./common";
-@import "./dismissible/mdc-drawer-dismissible";
-@import "./modal/mdc-drawer-modal";
+@import "@material/side-sheet/mixins";
+@import "@material/side-sheet/variables";
+@import "@material/typography/mixins";
+@import "./mixins";
+@import "./variables";
+
+// postcss-bem-linter: define drawer
```

Comment:
Curious, why is this annotation required?

Reply:
I think it enforces that each `.scss` file should only contain one top-level "block" class
(in this case, `mdc-drawer`).

Referencing any other block classes will throw an error, which is why we need a
`// postcss-bem-linter: ignore` comment on selectors like `.mdc-list-item:nth-child(1)`.